install/page-index/page-index.js

```diff
@@ -125,6 +125,15 @@ module.exports = [
         keywords: ["install", "Managed Service for TimescaleDB"],
         excerpt: "Install a managed service for TimescaleDB",
       },
+      {
+        title: "Kubernetes",
+        href: "installation-kubernetes",
+        title: "TimescaleDB on Kubernetes",
+        href: "installation-kubernetes",
+        tags: ["timescaledb", "install", "self-hosted", "kubernetes", "helm"],
+        keywords: ["install", "TimescaleDB", "self-hosted", "Kubernetes"],
+        excerpt: "Install TimescaleDB on Kubernetes",
+      },
```

Comment:
I don't have a huge opinion here, but seems like this makes sense to put after the "pre-built containers" section rather than at the bottom since they're closely related.

Reply:
@ryanbooz  moved as suggested.